internal/pkg/rpaas/k8s.go

```diff
@@ -125,6 +128,64 @@ func (q *fixedSizeQueue) Next() *remotecommand.TerminalSize {
 	return q.sz
 }
 
+func (m *k8sRpaasManager) Log(ctx context.Context, instanceName string, args LogArgs) error {
+	instance, err := m.GetInstance(ctx, instanceName)
+	if err != nil {
+		return err
+	}
+
+	nginx, err := m.getNginx(ctx, instance)
+	if err != nil {
+		return err
+	}
+
+	podsInfo, err := m.getPods(ctx, nginx)
+	if err != nil {
+		return err
+	}
+
+	var candidate v1.Pod
+	for _, pod := range podsInfo {
+		if pod.Name == args.Pod && pod.Status.Phase == "Running" {
+			candidate = pod
```

Comment:
@arthurcgc could you notify the user that have selected that pod ?, for example, some users may follow the behavior of tsuru apps, that receive all pods into a one command.

Example:
Write("Selected pod %s, if you want to check logs of other pods you should select the name of pod")
time.Sleep(time.Secod)

the code above could placed on client-side. If you want to inform the name of pod, you could use a header to carry, cause the body is a buffer.

Reply:
Yup, you're right... I'll add it to the command description and before the command execution. You're also right about the request body, it's already changed 😄